charts/fission-all/templates/pvc.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: fission-storage-pvc
+  labels:
+    app: fission-storage
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+  annotations:
+  {{- if .Values.persistence.storageClass }}
+    volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
```

Comment:
Should we use `storageClassName ` instead of annotation `volume.beta.kubernetes.io/storage-class`? 
According to the [doc](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1), the annotation will be deprecated in the future release.

Reply:
Thanks, didn't know that!

Reply:
Ah after reading this in more detail, it looks like leaving storageClassName unspecified is equivalent to choosing the default. As far as I can tell this is the only way to get a portable chart.